components/public-api/gitpod/experimental/v2/workspace.proto

```diff
@@ -0,0 +1,260 @@
+syntax = "proto3";
+
+package gitpod.experimental.v2;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v2";
+
+service WorkspaceService {
+  // GetWorkspace returns a single workspace.
+  //
+  // +return NOT_FOUND User does not have access to a workspace with the given
+  // ID +return NOT_FOUND Workspace does not exist
+  rpc GetWorkspace(GetWorkspaceRequest) returns (GetWorkspaceResponse) {}
+}
+
+message GetWorkspaceRequest { string id = 1; }
+
+message GetWorkspaceResponse { Workspace item = 1; }
```

Comment:
nit: Has there been a discussion on the use of `item` as generic name for the contained element in the response?
Alternatively, one could name it specific to the resource type (`workspace`).